apis/nucleus/src/components/listbox/interactions/listbox-keyboard-navigation.js

```diff
@@ -129,6 +130,9 @@ export function getListboxInlineKeyboardNavigation({
       case KEYS.ARROW_DOWN:
         updateKeyScrollOnHover({ down: 1 });
         break;
+      case KEYS.ARROW_LEFT:
+      case KEYS.ARROW_RIGHT:
+        return; // let it propagate to top-level
```

Comment:
this should not be needed as default is also return.
but maybe we still want it for better info